engine/apps/public_api/tests/test_custom_actions.py

```diff
@@ -269,94 +302,39 @@ def test_create_custom_action_valid_after_render_use_all_data(make_organization_
 
     response = client.post(url, data=data, format="json", HTTP_AUTHORIZATION=f"{token}")
 
-    custom_action = CustomButton.objects.get(public_primary_key=response.data["id"])
+    custom_action = Webhook.objects.get(public_primary_key=response.data["id"])
 
     expected_result = {
         "id": custom_action.public_primary_key,
         "name": custom_action.name,
         "team_id": None,
-        "url": custom_action.webhook,
+        "url": custom_action.url,
         "data": custom_action.data,
-        "user": custom_action.user,
+        "user": custom_action.username,
         "password": custom_action.password,
         "authorization_header": custom_action.authorization_header,
-        "forward_whole_payload": custom_action.forward_whole_payload,
+        "forward_whole_payload": custom_action.forward_all,
+        "is_webhook_enabled": custom_action.is_webhook_enabled,
+        "trigger_template": custom_action.trigger_template,
+        "headers": custom_action.headers,
+        "http_method": custom_action.http_method,
+        "trigger_type": Webhook.PUBLIC_TRIGGER_TYPES_MAP[custom_action.trigger_type],
+        "integration_filter": custom_action.integration_filter,
     }
 
     assert response.status_code == status.HTTP_201_CREATED
     assert response.json() == expected_result
 
 
-@pytest.mark.django_db
-def test_create_custom_action_invalid_data(
```

Comment:
This test was removed since validation is now processed at runtime because URLs can now contain templates.